src/wax_contract.cpp

```diff
@@ -108,18 +108,28 @@ CONTRACT WAX_CONTRACT_NAME: public contract {
                 &sig_val, sizeof(sig_val), random_value, sig_it->exponent, sig_it->modulus),
              "Could not verify signature.");
         
-        jobs_table.modify(job_it, get_self(), [&](auto& rec) {
-            rec.random_value = random_value;
-        });
-
-	checksum256 rv_hash = eosio::sha256(random_value.data(),
-          random_value.size());
+        checksum256 rv_hash = eosio::sha256(random_value.data(), random_value.size());
 
         action(
             {get_self(), "active"_n}, 
             job_it->caller, "receiverand"_n,
             std::tuple(job_it->assoc_id, rv_hash))
             .send();
+
+        jobs_table.erase(job_it);
+    }
+
+    /**
+     * @dev Removes jobs from the jobs table. The Oracle calls on it passing a list of dangling jobs.
+     * @param job_ids A vector of jobs IDs to be removed.
+     */
+    ACTION killjobs(const std::vector<int>& job_ids) {
+        for (const auto& e : job_ids) {
+            auto job_it = jobs_table.find(e);
+            check(job_it != jobs_table.end(), "Could not find job id.");
```

Comment:
I asked about this in the meeting: Should we assert if the job could not be found? IIRC we agreed to ignore the error.

Reply:
yeah I would agree. If the job exists then delete. if not, ignore